python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cleaning_strategy.py

```diff
@@ -71,7 +71,7 @@
 # .. math::
 #
 #   h(\boldsymbol{\xi}) = \sum_{\boldsymbol{\alpha} \in \mathcal{A}^{d}}
-#   a_{\boldsymbol{\alpha}} \psi_{\boldsymbol{\alpha}}(\boldsymbol{\xi}) + \epsilon.
+#   a_{\boldsymbol{\alpha}} \psi_{\boldsymbol{\alpha}}(\boldsymbol{\xi})
```

Comment:
I think that the suggested correction is wrong because the expansion is truncated up to the degree $d$ (as explained on lines 53-69). The consequence is that we cannot have equality between the expansion and the function. 

This is why the error was _not_ to write $+\epsilon$. The error was to write that $\epsilon$ is a random variable (which was written earlier), which is not.

I suggest to write:

$$
\widetilde{h}(\boldsymbol{\xi}) 
= \sum_{\boldsymbol{\alpha} \in \mathcal{A}^{d}}
a_{\boldsymbol{\alpha}} \psi_{\boldsymbol{\alpha}}(\boldsymbol{\xi})
$$

where $\widetilde{h}$ is an approximation of the $h$ function.

PS
One of the reasons I got confused at this stage is the equation 2.54 page 34 in [^LeMaître2010]. But the significant difference is that the left hand side of [^LeMaître2010] is a random variable. One of the potential sources of confusions is that the greek letter $\xi$ does not make clear the distinction with its random counterpart, which should write $\Xi$.

[^LeMaître2010]: Le Maître, O., & Knio, O. M. (2010). _Spectral methods for uncertainty quantification: with applications to computational fluid dynamics._ Springer Science & Business Media.